src/lib_c/x86_64-macosx-darwin/c/stdlib.cr

```diff
@@ -7,6 +7,8 @@ lib LibC
     rem : Int
   end
 
+  fun arc4random : UInt32
+  fun arc4random_buf(x0 : Void*, x1 : SizeT) : Void
```

Comment:
Please:
- Replace `Void*` with `UInt8*`
- Remove `: Void`

Reply:
Should this be the case even if it goes against the existing c definition?

```c
void
arc4random_buf(void *buf, size_t nbytes);
```

Reply:
I guess that works... I thought `Void*` didn't work in Crystal, but for some reason it does...

It seems there's also a `Void` top-level type... it seems it was never removed from the language...

Reply:
The `: Void` is implicit and the `void *` should be `UInt8*` in crystal. @asterite is correct.